ext/debug/ctf/README.rst

```diff
@@ -0,0 +1,216 @@
+.. _ctf:
```

Comment:
There are also a few reST formatting issues I could comment on, but since we don't include documentation in the ``ext`` folder in our published documentation it's won't show up as an error. This README will be here in the Git repo for folks to read if needed.

Reply:
I ran rstcheck and fixed a couple of issues.